01-messaging.md

```diff
@@ -5,7 +5,14 @@
 This protocol assumes an underlying authenticated and ordered transport mechanism that takes care of framing individual messages.
 [BOLT #8](08-transport.md) specifies the canonical transport layer used in Lightning, though it can be replaced by any transport that fulfills the above guarantees.
 
-The default TCP port is 9735. This corresponds to hexadecimal `0x2607`: the Unicode code point for LIGHTNING.<sup>[1](#reference-1)</sup>
+The default TCP port depending from the network used. The most common network are:
+
+- Bitcoin mainet with port number 9735 or the corresponds hexadecimal `0x2607`;
+- Bitcoin testnet with port number 19735  or the hexadecimal `0x4D17`;
+- Litecoin mainet with port number 9736 or the corresponds hexadecimal `0x2608`;
+- Litecoin testnet with port number 19736 or the corresponds hexadecimal `0x4D18`.
```

Comment:
and again, corresponding (could be dried DRY = Don't Repeat Yourself) by just including the hexadecimal in parentheses (after writing the full "or the corresponding hexadecimal" with no parentheses on the first line)

Reply:
Yes, it is a good idea, thanks!